docs/tools/lab/saved/keypairs.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: "Keypairs"
+sidebar_label: Keypairs
+description: "View saved keypairs (accounts)."
+sidebar_position: 0
+---
+
+# [Keypairs](https://lab.stellar.org/account/saved)
+
+## Overview
+
+![Lab: Saved Keypairs](/assets/lab/lab-account-saved.png)
+
+This page shows you saved keypairs for the selected network (only Testnet and Futurenet) in the browser's local storage. Every keypair has the following:
+
+1. Name - makes it easy to find the keypair you're looking for. You can update the name anytime by clicking the edit button and saving the new name.
+2. Public key - the public account address. You can quickly copy it with a click of a button.
+3. Secret key - the secret key of the account. Click the Eye button to toggle between masked and clear text format. Like the Public key, you can copy it by clicking the Copy button.
+4. Delete the saved keypair if it's no longer needed.
+5. The last saved date and time.
+6. XLM balance if the account is funded.
+7. If the account is unfunded, you can get 10,000 XLM by clicking the "Fund with Friendbot" button. This might be useful after the Testnet or Futurenet reset, as all your saved accounts will also be reset.
+
+## Manually save keypair
+
+To add a keypair manually, click the "Add keypair manually" button located in the top right corner of the screen. This will open a modal where you can enter a name for your keypair along with the [Stellar secret key](../../../learn/glossary.mdx#secret-private-key) (which begins with the letter "S") or recovery passphrase (a 12 or 24-word phrase).
+
```

Comment:
what do you think of adding an image or explanation that these keypairs can be used for `loading account` and `signing` in build transaction